docs/users/extend/customization-extendibility.md

```diff
@@ -55,3 +55,43 @@ To load the new configuration, run `ddev restart`.
 
 ## Overriding default container images
 The default container images provided by ddev are defined in the `config.yaml` file in the `.ddev` folder of your project. This means that _defining_ an alternative image for default services is as simple as changing the image definition in `config.yaml`. In practice, however, ddev currently has certain expectations and assumptions for what the web and database containers provide. At this time, it is recommended that the default container projects be referenced or used as a starting point for developing an alternative image. If you encounter difficulties integrating alternative images, please [file an issue and let us know](https://github.com/drud/ddev/issues/new).
+
+## Adding environment variables
```

Comment:
ddev is used for many different CMSs, dotenv-connector is only for TYPO3, right? That implies that we have to find a better place for this. Also, we're hoping for a much more general solution in [Make an easy way to populate environment variables in web container](https://github.com/drud/ddev/issues/757).

Reply:
dotenv is for any composer based project - not TYPO3 related